src/js/messaging/components/compose/MessageSend.jsx

```diff
@@ -62,14 +62,15 @@ class MessageSend extends React.Component {
 
   render() {
     const isDisabled = this.props.disabled;
+    const spinnerIcon = this.props.sendingMessage && <span className="fa fa-spinner fa-spin"/>;
 
     return (
       <div className="msg-send-group">
         <div className="msg-send-buttons">
           <button
             disabled={isDisabled}
             type="button"
-            onClick={this.props.onSend}>Send</button>
+            onClick={this.props.onSend}>{spinnerIcon} Send</button>
```

Comment:
Is this always going to have a space before the "Send" if it's not currently sending?

Reply:
Good question, I know it doesn't in Chrome but I need to confirm for other browsers

Reply:
Just did a little digging - [leading spaces are trimmed](https://developer.mozilla.org/en-US/docs/Web/CSS/white-space) from HTML elements unless there is special style applied like in the `pre` tag or you use `nbsp;`. So we should be good here.